01.md

```diff
@@ -168,8 +169,11 @@ This NIP defines no rules for how `NOTICE` messages should be sent or treated.
   * `["OK", "b1a649ebe8...", false, "invalid: event creation date is too far off from the current time"]`
   * `["OK", "b1a649ebe8...", false, "pow: difficulty 26 is less than 30"]`
   * `["OK", "b1a649ebe8...", false, "error: could not connect to the database"]`
-- `CLOSED` messages MUST be sent in response to a `REQ` when the relay refuses to fulfill it. It can also be sent when a relay decides to kill a subscription on its side before a client has disconnected or sent a `CLOSE`. This message uses the same pattern of `OK` messages with the machine-readable prefix and human-readable message. Some examples:
+
+- `CLOSED` messages MUST be sent in response to a `REQ` when the relay refuses to fulfill it, or in response to `CLOSE` to acknowledge the end of a subscription based on client request. It can also be sent when a relay decides to kill a subscription on its side before a client has disconnected or sent a `CLOSE`. This message uses the same pattern of `OK` messages with the machine-readable prefix and human-readable message. Some examples:
   * `["CLOSED", "sub1", "unsupported: filter contains unknown elements"]`
   * `["CLOSED", "sub1", "error: could not connect to the database"]`
   * `["CLOSED", "sub1", "error: shutting down idle subscription"]`
-- The standardized machine-readable prefixes for `OK` and `CLOSED` are: `duplicate`, `pow`, `blocked`, `rate-limited`, `invalid`, and `error` for when none of that fits.
+  * `["CLOSED", "sub1", "ok: sub1 closed successfully"]`
```

Comment:
I feel like `["CLOSED", "sub1"]` or `["CLOSED", "sub1", ""]` is enough.

Reply:
@AsaiToshiya yes, agree. based on my changes, your examples will be valid as well. let me use your examples.

Reply:
example updated, `ok` got removed.